src/utils/editor-utils.ts

```diff
@@ -0,0 +1,33 @@
+import * as path from 'path';
+import { DEFAULT_EDITORS, DefaultEditorId, EditorId } from '../interfaces';
+
+export function isKnownFile(filename: string): boolean {
+  return DEFAULT_EDITORS.includes(filename as any);
+}
+
+const TITLE_MAP: Record<DefaultEditorId, string> = Object.freeze({
+  [DefaultEditorId.main]: `Main Process (${DefaultEditorId.main})`,
+  [DefaultEditorId.renderer]: `Renderer Process (${DefaultEditorId.renderer})`,
+  [DefaultEditorId.preload]: `Preload (${DefaultEditorId.preload})`,
+  [DefaultEditorId.html]: `HTML (${DefaultEditorId.html})`,
+  [DefaultEditorId.css]: `Stylesheet (${DefaultEditorId.css})`,
+});
+
+export function getEditorTitle(id: EditorId): string {
+  return TITLE_MAP[id] || `Custom Editor (${id})`;
+}
+
+const EMPTY_EDITOR_CONTENT = {
+  css: '/* Empty */',
+  html: '<!-- Empty -->',
+  js: '// Empty',
+} as const;
```

Comment:
curiosity question: why are we adding `as const` here and elsewhere?

Reply:
Mostly just cleanliness. Everything that can be const'ed should be, so that the compiler finds the humans' mistakes :robot: